spec.html

```diff
@@ -1273,6 +1273,17 @@ <h1>Well-Known Symbols</h1>
                 An object valued property whose own and inherited property names are property names that are excluded from the `with` environment bindings of the associated object.
               </td>
             </tr>
+            <tr>
+              <td>
+                <dfn>@@metadata</dfn>
+              </td>
+              <td>
+                *"Symbol.metadata"*
+              </td>
+              <td>
+                An object valued property that contains the metadata associated with an object. Primarily used to store metadata defined by decorators on classes.
+              </td>
+            </tr>
```

Comment:
I believe the table is sorted alphabetically so `@@metadata` should be between `@@matchAll` and `@@replace`.